vscode-typescript/config.yml

```diff
@@ -0,0 +1,7 @@
+version: "2"
+containers:
+- name: container
+  image: codercom/code-server
+  cmd: code-server workspace /app --auth none --disable-telemetry
```

Comment:
Could we make the code-server operate in dark mode by default?

You could use the `--user-data-dir` flag for this and pass a config file at `user-data/User/settings.json`

```suggestion
  cmd: code-server workspace /app --user-data-dir user-data --auth none --disable-telemetry
```

Example configuration:

```
{
    "workbench.colorTheme": "Default Dark+",
    "workbench.statusBar.visible": false,
    "workbench.tips.enabled": false,
    "workbench.startupEditor": "newUntitledFile",
    "update.showReleaseNotes": false,
    "telemetry.enableTelemetry": false,
    "window.menuBarVisibility": "hidden",
    "workbench.activityBar.visible": false,
    "breadcrumbs.enabled": false,
    "breadcrumbs.filePath": "off",
    "workbench.editor.enablePreview": false,
    "workbench.editor.enablePreviewFromQuickOpen": false,
    "workbench.editor.highlightModifiedTabs": true,
    "workbench.editor.showTabs": false,
    "files.enableTrash": false,
    "workbench.enableExperiments": false,
    "editor.codeLens": false,
    "editor.colorDecorators": false,
    "editor.dragAndDrop": false,
    "editor.smoothScrolling": true,
    "editor.minimap.enabled": false,
    "editor.fontSize": 13,
    "workbench.colorCustomizations": {
        "sideBar.background": "#252F42",
        "sideBar.foreground": "#fff",
        "editor.background": "#1C2639",
        "list.activeSelectionBackground": "#344156",
        "list.focusBackground": "#344156",
        "list.hoverBackground": "#13171e",
        "list.inactiveSelectionBackground": "#344156"
    }
}
```

Reply:
Updated: https://github.com/instruqt/templates/pull/3/commits/0e935cb74ae4be4de46c24fd8557eca5916217f4.